### ISSUE TITLE
DLA Template VAC Review (DR1)

### ISSUE THREAD

**Issue by dylanagreen** [desihub/desida#30]
Contact Person: Allyson Brodzeller
1 catalog file (fits), 1 README

Initial Checks:
- [x] Includes README
- [x] Columns in ALLCAPS
- [x] Extension names in ALLCAPS
- [x] Files include units (see below)

Initial Notes:
- File is looking good so far! Can you please add units of degrees to RA and DEC columns? 
- README data model looks good, can you please add units to the data model where applicable?
- When you get a chance, can you please send a paragraph or two summary of the VAC to include in the DR1 paper?

**Comment by dylanagreen** [comment]
@weaverba137 Let's move this to final review with the stubname `dla-toolkit` and version `v1` (this will require renaming the file from v0 -> v1, but I've gotten permission from Allyson to do this). We also need not include the `old` directory, as it only contains older versions of the VAC for review purposes.

**Comment by weaverba137** [comment]
We definitely want `v1`, not `v1.0`?

**Comment by dylanagreen** [comment]
We can do v1.0, no opposition to that.

**Comment by weaverba137** [comment]
The files are now removed to the staging area. The original `README.md` file is retained temporarily as `README.md.bak`.

The only minor issue with the FITS file is no HDU-level checksum headers, but I don't think we need to delay further for that.

**Comment by dylanagreen** [comment]
We've merged the README for this and the files in the staging location look good to me, I say let's close the loop and move this to the final location @weaverba137, unless we want to require the checksum headers.

**Comment by weaverba137** [comment]
Done, close when ready.

**Comment by abrodze** [comment]
After chatting with @dylanagreen, I removed ~1000 from the catalog that should not have made the final cut. VAC is still at 
/global/cfs/cdirs/desi/users/abrodze/DLA/catalogs/DR1

**Comment by dylanagreen** [comment]
Yes, let's reopen this just until we can move the updated file to the staging and final areas on disk. It passes all of my checks.

**Comment by weaverba137** [comment]
I'll take a look at the updated file momentarily.

**Comment by weaverba137** [comment]
The new file looks fine. I've replaced the old file in the public area. If everything looks good, go ahead and close.

**Comment by dylanagreen** [comment]
Great, thanks @weaverba137!

**Comment by weaverba137** [comment]
Reopening this issue to discuss a revised version of this VAC that is still DR1-based.

**Comment by weaverba137** [comment]
I have staged the v2.0 catalogs to `${DESI_ROOT}/vac/dr1/dla-toolkit/v2.0`. Please take a look. See also desihub/desidatadocs#173.
